Formula/pam-u2f.rb

```diff
@@ -26,8 +25,13 @@ class PamU2f < Formula
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "libfido2"
-  depends_on "libu2f-host"
-  depends_on "libu2f-server"
```

Comment:
Removed what seem to be unused dependencies.
`libfido2` should have replaced `libu2f...` dependencies in previous update
https://github.com/Yubico/pam-u2f#building
> Versions of this project up to 1.0.8 used `libu2f-host` and `libu2f-server`

Reply:
On side note, `libu2f-host` should be marked deprecated as https://github.com/Yubico/libu2f-host is archived/unmaintained.

`libu2f-server` is still available but in maintenance mode: https://github.com/Yubico/libu2f-server